### DISCUSSION TITLE
I can't see my docx files in obsidian

### DISCUSSION THREAD

**Discussion by matar3** [dream-num/obsidian-univer#89] [General]
I don't know why, but i can correctly see and also open and edit the xlsx files in obsidian (through the univer plugin), but still can't see the docx files in the vault, as if they were invisible (but i'm sure they are there since i can see them in windows file explorer). Any suggestion to solve that?
Thank you

P.S. This would be an issue, but in the issue section i found only the "Feature request" template. Sorry if i've done sthg wrong.

**Comment by jikkai** [comment]
We currently do not support docx or pptx files.

**Reply by matar3** [reply]
Ah, ok. I thought i was doing sthg wrong since the description of the plugin says that the purpose is to create, view and edit documents like excel and word. I hope you'll add this feature in the future.
